lms/services/exceptions.py

```diff
@@ -158,6 +158,10 @@ class OAuth2TokenError(ExternalRequestError):
     """
 
 
+class ConflictError(Exception):
+    """A request failed due to a conflicting operation."""
```

Comment:
The reason for having this error is to capture a low-level error which might be used in multiple contexts (acquiring an advisory DB lock failed) and mapping it to a more contextual error (there was a conflict which caused the refresh operation to fail). I've realized that on the frontend it might be useful to know the specific error so that a more friendly message can be shown in the event that a concurrent refresh does eventually time out. So I'm planning to change this to `ConcurrentTokenRefreshError`.